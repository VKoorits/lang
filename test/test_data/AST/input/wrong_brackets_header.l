sub factorial n:
	if n < 2:
		return 1
	return b*factorial(n-1)
print("ok")
