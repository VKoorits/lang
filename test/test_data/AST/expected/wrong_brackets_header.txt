ERROR: second and last token must be brackets
